src/hooks/useCreateTicketMultiAttestations.ts

```diff
@@ -0,0 +1,82 @@
+import { useSigner } from "@/hooks/useSigner";
+import { encodePacked, keccak256, zeroAddress } from "viem";
+import { EAS, SchemaEncoder } from "@ethereum-attestation-service/eas-sdk";
+import { useCallback, useEffect, useState } from "react";
+import type { Address } from "viem";
+
+const CREATE_TICKET_SCHEMA = "address owner, string eventId";
+
+const CREATE_TICKET_SCHEMA_UID = keccak256(
+  encodePacked(
+    ["string", "address", "bool"],
+    [CREATE_TICKET_SCHEMA, zeroAddress, true]
+  )
+);
+
+export interface Ticket {
+  owner: Address;
+  eventId: string;
+}
+
+export const CREATE_TICKET_SCHEMA_ENCODER = new SchemaEncoder(
+  CREATE_TICKET_SCHEMA
+);
+
+export const useCreateTicketMultiAttestations = () => {
+  const EAS_CONTRACT_ADDRESS = "0x4200000000000000000000000000000000000021";
+
+  const signer = useSigner();
+  const [eas, setEas] = useState<EAS | null>(null);
+
+  useEffect(() => {
+    if (signer) {
+      const newEas = new EAS(EAS_CONTRACT_ADDRESS);
+      newEas.connect(signer);
+      setEas(newEas);
+    }
+  }, [signer]);
+
+  console.log(CREATE_TICKET_SCHEMA_UID);
```

Comment:
This console.log should be here? If it's just for testing, you should add a TODO: comment just to remember to remove it.